src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: CSS Grid is a new layouting system for the web.
+
+
+{# wf_published_on: 2017-01-27 #}
+{# wf_updated_on: 2017-01-27 #}
+{# wf_featured_image: /web/updates/images/2017/01/css-grid/grid.png #}
+{# wf_tags: css,layout,grid #}
+{# wf_featured_snippet: CSS Grid is a new layouting system for the web. #}
```

Comment:
Suggestion (not required): `{# wf_featured_snippet #}` is what's used on the updates landing page and is your opportunity to hook people. I'd use something like:

CSS Grid Layout makes creating two dimensional grid based layouts easy. It's been in development for over 5 years, but is now available in Chrome and coming to other browsers soon. Let's take a peek at what's new and how you can use it on your sites!